client/extensions/wp-job-manager/components/setup/page-setup/index.jsx

```diff
@@ -0,0 +1,174 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { formValueSelector, reduxForm } from 'redux-form';
+import { localize } from 'i18n-calypso';
+import { flowRight as compose } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CompactCard from 'components/card/compact';
+import ExternalLink from 'components/external-link';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+import ReduxFormToggle from 'components/redux-forms/redux-form-toggle';
+import SectionHeader from 'components/section-header';
+
+const form = 'extensions.wpJobManager.pageSetup';
+
+class PageSetup extends Component {
+	static propTypes = {
+		createDashboard: PropTypes.bool,
+		createJobs: PropTypes.bool,
+		createPostJob: PropTypes.bool,
+		translate: PropTypes.func.isRequired,
+	};
+
+	render() {
+		const {
+			createDashboard,
+			createJobs,
+			createPostJob,
+			translate,
+		} = this.props;
+
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Page Setup' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Page setup:' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).